### DISCUSSION TITLE
Adjust calculation behavior

### DISCUSSION THREAD

**Discussion by snhv** [Qalculate/libqalculate#388] [Q&A] [answered]
Currently, Qalculate understands
1÷10+1% as 1÷10 x 101%
**not** 1÷10+0.01 which is what I want.
Is there any way to change this behavior?
Thanks.

**Comment by foreachthing** [comment]
Same result from [goolge](https://www.google.com/search?q=1%C3%B710%2B1%25) and [wolframalpha](https://www.wolframalpha.com/input/?i=1%2F10%2B1%25).
So, **no**, please don't change anything.

**Reply by snhv** [reply]
> Same result from [goolge](https://www.google.com/search?q=1%C3%B710%2B1%25) and [wolframalpha](https://www.wolframalpha.com/input/?i=1%2F10%2B1%25). So, **no**, please don't change anything.

Different result from excel.
I'm not asking for a change in default behavior. I'm asking if i can adjust this in the program for my own use. That's why this thread is marked as a Q&A.

**Reply by foreachthing** [reply]
My apologies, I didn't see it marked as Q&A. 🙈 
That's because Excel treats 1% as 0.01. No matter what. You should then type (in Qalculate): `1÷10+0.01`.

**Comment by hanna-kn** [comment, marked as answer]
It is not possible to change the behaviour in the latest released version (primarily because of the limited use for ordinary addition/subtraction of percent), but I've now added an option (which will be included in the next version, planned for 2022-02-14) to change the default behaviour.

**Reply by snhv** [reply]
Thank you very much. I look forward to the new version. I've been accustomed to how Excel and Casio handheld calculators work for too long.

**Comment by hanna-kn** [comment]
It should be possible to bypass the default behaviour by deactivating the default percent constant and replace it with a new, although I do not recommend it.

**Reply by snhv** [reply]
Hi @hanna-kn 
I've just installed the new 4.0 version. But I don't know how to disable the so-called simplified percentage calculation. Could you help me please?
Thank you.

**Reply by hanna-kn** [reply]
In qalc `set percent 0` (or `-s "percent 0"` on the command line), in qalculate-gtk Mode->Parsing Mode->Simplified Percentage Calculation, in qalculate-qt Preferences->Parsing & Calculation->Simplified Percentage Calculation.

**Reply by snhv** [reply]
Got it. Thank you so much!
